### PR TITLE
ankush dohare

### PR DESCRIPTION
**Pull request by xankush**
thanks for this opportunity, I added some basic java code

### CODE REVIEW

File: java/forloop.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class forloop{
+    public static void main(string[] args){
+        //syntex for for loop
+        //for (initilize;condition;update)
+        for(int n=0;n<=10;n++){
+        System.out.println("hello world");
+        }
```
Comment (mr-ayush-agrawal) [inline java/forloop.java:7]: This code is just Printing a statement over a loop

File: java/howtoprint.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+public class howtoprint{
+    public static void main(string[] args){
+        System.out.println("type here what you want to print");
+        //shortcut for system.out.println();
+        // type sout then hit tab button
```
Comment (mr-ayush-agrawal) [inline java/howtoprint.java:5]: Only a single print statement

File: java/scanner.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import java.util.*;
+public class scanner {
+    public static void main(String[] args){
+        Scanner sc = new Scanner(System.in);
+        String a = sc.next();
+        System.out.println(a);
+
```
Comment (mr-ayush-agrawal) [inline java/scanner.java:7]: Just Input and output

**Comment by mr-ayush-agrawal** [comment]
I have reviewed the codes these are just some one line input output statements.
I can't merge these codes.

Make some upgradation such that these may add value to repo.

**Comment by xankush** [comment]
Thanks for the feedback, I will do some upgrade ,I will try my best

**Pull request closed without merging**